docs/proving-grounds/html-example-list/README.md

```diff
@@ -0,0 +1,157 @@
+---
+navbar: false
+pageClass: cdr-doc-proving-ground
+---
+
+# Html Example List Proving Grounds
+
+## Default Component Behavior
+<cdr-doc-html-example-list>
+```html
+  <div>
+    <cdr-button size="large">Button 1</cdr-button>
+    <cdr-button size="small">Button 2</cdr-button>
+    <cdr-button>Button 3</cdr-button>
+  </div>
+```
+</cdr-doc-html-example-list>
+
+## With Label
+<cdr-doc-html-example-list label="State of component">
+```html
+  <div>
+    <cdr-button size="large">Button 1</cdr-button>
+    <cdr-button size="small">Button 2</cdr-button>
+    <cdr-button>Button 3</cdr-button>
+  </div>
+```
+</cdr-doc-html-example-list>
+
+## No Background Toggle
+<cdr-doc-html-example-list :background-toggle="false">
+```html
+  <cdr-list
+    modifier="unordered"
+    class="cdr-stack--lg"
+  >
+    <li>List item text</li>
+    <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmodList ordered
+      <cdr-list>
+        <li>List item text</li>
+        <li>List item text
+          <cdr-list>
+            <li>List item text</li>
+            <li>List item text</li>
+          </cdr-list>
+        </li>
+      </cdr-list>
+    </li>
+    <li>List item text</li>
+  </cdr-list>
+```
+</cdr-doc-html-example-list>
+
+## No Background Toggle, background color set to dark
+<cdr-doc-html-example-list :background-toggle="false" background-color="dark">
+```html
+  <cdr-button>Testing</cdr-button>
+```
+</cdr-doc-html-example-list>
+
+## Multiple Examples
+<cdr-doc-html-example-list backgroundColor>
+  <template slot="Default">
+
+```html
+  <cdr-button size="large">Default</cdr-button>
+```
+  
+  </template>
+  <template slot="Hover">
+
+```html
+  <cdr-button size="small" class="cdr-doc-button-demo--hover">Hover</cdr-button>
```

Comment:
I'm not a fan of maintaining doc classes that set false states to component interactions, at best they provide duplicative experience to the actual component, at worst we have something that now has to be versioned or at least maintained prior to an update that has not proven to be helpful to the consumer

Reply:
This is an example of how the component could be used to show states of various components. This component handles the display of multiple examples, and the proving ground is not a prescription for how or even if it should be used this way.